### DISCUSSION TITLE
overwrite publish library with gradle

### DISCUSSION THREAD

**Discussion by manang** [community/community#37101] [Actions]
Hi,
I'm trying to avoid the problem of publishing a new java library with gradle, if the same library, with the same version, is present on github maven repo.
I receive the error "Received status code 409 from server: Conflict"
do you have any idea to manage it?
Thanks a lot
Angelo

**Comment by ossanna16** [comment]
Hi there @manang and welcome to our community! Thank you for asking a great question 🙂
